### ISSUE TITLE
gestures not working if start point on textbox

### ISSUE THREAD

**Issue by shinji2009** [gomita/firegestures#123]
subj. gestures not working if start point is on textbox.
also not working on ebay in the bottom of any product page where Specifications descriptions photos shippings etc
latest ff

**Comment by gomita** [comment]
If your 'Trigger button to start a gesture' option is 'left', it's normal behavior.

**Comment by shinji2009** [comment]
no, its right. and it not was always like this i think it started from some version of ff...
